packages/hash/frontend/src/pages/type-editor/blockprotocol-ontology-functions-hook.ts

```diff
@@ -100,17 +98,12 @@ export const useBlockProtocolFunctionsWithOntology = (
   const { getEntityType } = useBlockProtocolGetEntityType();
   const { updateEntityType } = useBlockProtocolUpdateEntityType(readonlyMode);
 
-  const { updateLink } = useBlockProtocolUpdateLink();
-
   return {
     aggregateEntities,
     createEntity,
```

Comment:
I'm getting an error here: `'CreateEntityMessageCallback' is not assignable to type 'MessageCallback<CreateEntityData, null, Entity<Record<string, unknown>>, CreateResourceError>'`
Is this sorted in the PR that fixes the hooks? I suppose it's because of the `@blockprotocol/graph` types

Reply:
I encountered this too. It's fixed in one of the upcoming PRs, I can't quite remember which. But we'll be ensuring TSC passes (which it doesn't currently), so I think it's fine to leave this for now. It's not a regression as such.